python/paddle/nn/functional/common.py

```diff
@@ -2081,67 +2082,64 @@ class centers and the shape of sampled_class_center will be [num_positive_class_
     .. code-block:: python
         :name: code-example1
 
-        # CPU or single GPU
-        import paddle
-        num_classes = 20
-        batch_size = 10
-        num_samples = 6
-        label = paddle.randint(low=0, high=num_classes, shape=[batch_size], dtype='int64')
-        remapped_label, sampled_class_index = paddle.nn.functional.class_center_sample(label, num_classes, num_samples)
-
-        print(label)
-        print(remapped_label)
-        print(sampled_class_index)
-
-        # the output is
-        #Tensor(shape=[10], dtype=int64, place=CPUPlace, stop_gradient=True,
-        #       [11, 5 , 1 , 3 , 12, 2 , 15, 19, 18, 19])
-        #Tensor(shape=[10], dtype=int64, place=CPUPlace, stop_gradient=True,
-        #       [4, 3, 0, 2, 5, 1, 6, 8, 7, 8])
-        #Tensor(shape=[9], dtype=int64, place=CPUPlace, stop_gradient=True,
-        #       [1 , 2 , 3 , 5 , 11, 12, 15, 18, 19])
+        >>> # CPU or single GPU
+        >>> import paddle
+        >>> num_classes = 20
+        >>> batch_size = 10
+        >>> num_samples = 6
+        >>> label = paddle.randint(low=0, high=num_classes, shape=[batch_size], dtype='int64')
```

Comment:
- `paddle.randint` 也是带有随机性的，加一个随机种子吧

```suggestion
        >>> paddle.seed(2023)
        >>> label = paddle.randint(low=0, high=num_classes, shape=[batch_size], dtype='int64')
```

- 下面的输出，也记得按照seed的结果同步修改